README.md

```diff
@@ -1,24 +1,199 @@
 
-* Deployment instructions
+## Live Demo & Presentation <a name="live-demo">
+[LIVE DEMO](https://cash-in-out-qoka.onrender.com/)
 
-* ...
+<!-- Video Presentation -->
+## Video Presentation <a name="video-presentation"></a>
+[See Video](https://vimeo.com/883597169/4195c3d8ef?share=copy)
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> 
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add categories**
+- **Add transactions**
+- **Mobile friendly UI**
+- **Delete Account**
+- **Topup credit**
+- **Clear credit**
+- **Balance display**
+- **User Authentication**
+- **Splash screen**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+- Code editor like VS code
+- Ruby installed in your machine
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+   git clone https://github.com/geekelo/cash-in-out.git
+   cd cash-in-out
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd cash-in-out
+  gem install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```sh
+  rails server
+```
+
+### Run Tests
+To run tests, run the following command:
+```
+  bin/rails test test/models/article_test.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Eloghene Otiede**
+
+- GitHub: [@githubhandle](https://github.com/geekelo)
+- Twitter: [@twitterhandle](https://twitter.com/geekelo_xyz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/eloghene-otiede)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] Email Notifications
+- [ ] Animations and transitions
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/geekelo/unibek-school_library_app/issues).
```

Comment:
- [ ] I noticed that the issues page led to a totally different repo. KIndly fix this link with the correct one.